ts/Core/Chart/ChartOptions.d.ts

```diff
@@ -95,6 +95,7 @@ export interface ChartOptions {
     borderWidth?: number;
     className?: string;
     colorCount?: number;
+    createDataCoppy?: boolean;
```

Comment:
Typo `coppy`

Reply:
I'm not sure if it's a good name. It says something about how we deal with the mutation problem internally, but is not immediately recognizable to the implementer. The original issue report is called "setData mutating source array". I think a more descriptive property name would be something along the lines of `chart.allowMutatingData`. Which would in turn mean the polarity would be reversed and `true` be default.